samples/project-anfield/README.md

```diff
@@ -162,6 +162,30 @@ This will work for Python 3 >=3.4.
 12. IMPORTANT: If any password or secretkey in the config file have special character **%**, replace __%__ with __%%__ as special character(%) is considered as escape character in python ConfigParser.
 For example, if your password is **Cohesity%123** provide in the config file as **Cohesity%%123**
 
+## Encrypt
+
+Run the following command to encrypt the default configuration file(config.ini).
+```
+python crypt.py encrypt -k key_to_encrypt
+```
+Run the following command to encrypt the custom configuration file provided.
+```
+python crypt.py encrypt -c /path/to/config/file -k key_to_encrypt
```

Comment:
python secure_config_file.py -operation encrypt -c /path/to/config.ini -key key_to_encrypt